src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java

```diff
@@ -267,67 +281,111 @@ void testSaveCommentsForGoalByIdException() {
 
     @Test
     @DisplayName("Test isCurrentUsersGoal(Long goalId): verify if goal is for current user is returned true")
-    void isCurrentUsersGoalTestForCurrentUser() {
+    void testIsCurrentUsersGoalForCurrentUser() {
         when(userStore.getCurrentUser())
                 .thenReturn(user1);
-        when(goalStore.getUserGoalById(user1, 1L))
+        when(goalStore.getUnachievedUserGoalById(user1, 1L))
                 .thenReturn(Optional.of(goal));
         assertThat(goalApi.isCurrentUsersGoal(1L), is(true));
     }
 
     @Test
     @DisplayName("Test isCurrentUsersGoal(Long goalId): verify if goal is for other user not current, then returned false")
-    void isCurrentUsersGoalTestForDifferentUser() {
+    void testIsCurrentUsersGoalForDifferentUser() {
         when(userStore.getCurrentUser())
                 .thenReturn(user2);
-        when(goalStore.getUserGoalById(user2, 1L))
+        when(goalStore.getUnachievedUserGoalById(user2, 1L))
                 .thenReturn(Optional.empty());
         assertThat(goalApi.isCurrentUsersGoal(1L), is(false));
     }
 
     @Test
     @DisplayName("Test editGoal(Long goalId, GoalFormDto newGoalDto): verify that goal is edited, if user created goal")
     void testEditGoalForGoalOwner() {
+        goal.setStatus(GoalStatus.OPEN);
         GoalFormDto goalDto = new GoalFormDto();
```

Comment:
Please extract into separate method here and in places of this class where it is relevant. It's a code duplication